microsoft/PowerToys
# [FileLockSmith] Show process files in modal dialog

## Summary of the Pull Request
To prevent a crash, show the individual files of a process within a modal dialog rather than as inline text. Please see the linked issue for more details.

## PR Checklist

- [X] **Closes:** #32663

## Detailed Description of the Pull Request / Additional comments
- Removed inline list of process files.
- Added "Show files" button to expander to show list of process files as a modal dialog. This dialog has the same design as the one used to display the list of selected folders within the same application.
- Added unhandled exception hander to application similar to our other applications.

![image](https://github.com/user-attachments/assets/52eddfcc-5e10-40a3-94b2-68bbfb607f1d)

![image](https://github.com/user-attachments/assets/ff996e32-36f6-41a9-a9f0-6dda7a93d09a)

## Validation Steps Performed
- Tested change.
- Tested app no longer crashes after bringing up dialog several times.
- Sanity check of app.
